watchmal/dataset/cnn/cnn_dataset.py

```diff
@@ -26,31 +25,40 @@ def __init__(self, h5file, pmt_positions_file, is_distributed, transforms=None,
         super().__init__(h5file, is_distributed)
         
         self.pmt_positions = np.load(pmt_positions_file)['pmt_image_positions']
+        self.use_times = use_times
+        self.use_charges = use_charges
         self.data_size = np.max(self.pmt_positions, axis=0) + 1
         self.barrel_rows = [row for row in range(self.data_size[0]) if
                             np.count_nonzero(self.pmt_positions[:,0] == row) == self.data_size[1]]
-        self.data_size = np.insert(self.data_size, 0, 1)
+        self.data_size = np.insert(self.data_size, 0, 2)
```

Comment:
I think if you have a configuration option for whether you're using times, charges or both, the size of the data should change depending on whether or not you're using both.

Reply:
I made the changes. The user still has to set 'num_input_channels' in the model config to match. I believe this is also the case with the existing pointnet class and the 'k' parameter.